app/main.py

```diff
@@ -1,8 +1,32 @@
+from typing import List, Dict, Any
+
+
 class Person:
-    # write your code here
-    pass
+    people: Dict[str, "Person"] = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
+
+
+def create_person_list(people: List[Dict[str, Any]]) -> List[Person]:
```

Comment:
Mesma coisa aqui! Sua resposta não está errada, mas o ideal mesmo é que, como a função já estava declarada, não troquemos a declaração dela. Pode deixar da maneira que estava.

```suggestion
def create_person_list(people: list) -> list:
```